packages/core/src/types/authMachine.ts

```diff
@@ -26,12 +51,13 @@ export type AuthEventTypes =
   | 'SIGN_OUT'
   | 'SUBMIT'
   | 'RESEND'
-  | 'CONFIRM_SIGN_UP'
-  | 'CONFIRM_SIGN_IN'
-  | 'INPUT'
   | 'CHANGE'
   | 'FEDERATED_SIGN_IN'
-  | 'RESET_PASSWORD';
+  | 'RESET_PASSWORD'
+  | 'done.invoke.signInActor'
+  | 'done.invoke.signUpActor'
+  | 'done.invoke.signOutActor'
+  | 'done.invoke.resetPasswordActor';
```

Comment:
nit - since these are events of a different "format" than the previous `AuthEventTypes`, can we make these into their own `type`? Something like `export type InvokeActorEventTypes`. We can then either extend from `AuthEventTypes`, or just change where these types are used to except either `AuthEventTypes` or `InvokeActorEventTypes`.

Just a thought. Not blocking over it.

Reply:
I like this idea! Done here -- https://github.com/aws-amplify/amplify-ui/pull/180/commits/fbed3464311ca46b93e7b7bb498ce8e242486bad